grpc/src/main/java/io/stargate/grpc/service/Service.java

```diff
@@ -563,6 +731,20 @@ private Parameters makeParameters(QueryParameters parameters) {
     return builder.tracingRequested(parameters.getTracing()).build();
   }
 
+  private Parameters makeTracingParameters(QueryParameters parameters) {
+    ImmutableParameters.Builder builder = ImmutableParameters.builder();
+    // only consistency levels can be inherited by tracing query
+    if (parameters.hasConsistency()) {
```

Comment:
I'm not sure it makes sense to inherit the consistency levels from the original queries for the tracing queries. For example the java-driver uses a default consistency of ONE for tracing queries. 

https://github.com/datastax/java-driver/blob/4.x/core/src/main/java/com/datastax/oss/driver/api/core/config/OptionsMap.java#L293